examples/tutorials/sst2_classification_non_distributed.py

```diff
@@ -0,0 +1,156 @@
+DEV_DATAPIPE = DEV_DATAPIPE.batch(BATCH_SIZE)
+DEV_DATAPIPE = DEV_DATAPIPE.rows2columnar(["token_ids", "target"])
+
+# Alternately we can also use batched API (i.e apply transformation on the whole batch)
+# TRAIN_DATAPIPE = TRAIN_DATAPIPE.batch(BATCH_SIZE).rows2columnar(["text", "label"])
+# TRAIN_DATAPIPE = TRAIN_DATAPIPE.map(lambda x: {"token_ids": TEXT_TRANSFORM(x["text"]), "target": LABEL_TRANSFORM(x["label"])})
+# DEV_DATAPIPE = DEV_DATAPIPE.batch(BATCH_SIZE).rows2columnar(["text", "label"])
+# DEV_DATAPIPE = DEV_DATAPIPE.map(lambda x: {"token_ids": TEXT_TRANSFORM(x["text"]), "target": LABEL_TRANSFORM(x["label"])})
+
+######################################################################
+# Model Preparation
+# -----------------
+# TODO
+NUM_CLASSES = 2
+INPUT_DIM = 768
+
+from torchtext.models import RobertaClassificationHead, XLMR_BASE_ENCODER
+classifier_head = RobertaClassificationHead(num_classes=NUM_CLASSES, input_dim=INPUT_DIM)
+MODEL = XLMR_BASE_ENCODER.get_model(head=classifier_head)
+MODEL.to(DEVICE)
+
+
+#######################################################################
+# Training methods
+# ----------------
+# TODO
+
+import torchtext.functional as F
+from torch.optim import AdamW
+
+LEARNING_RATE = 1e-5
+OPTIM = AdamW(MODEL.parameters(), lr=LEARNING_RATE)
+CRITERIA = nn.CrossEntropyLoss()
+
+
+def train_step(input, target):
+    output = MODEL(input)
+    loss = CRITERIA(output, target)
+    OPTIM.zero_grad()
+    loss.backward()
+    OPTIM.step()
+
+
+def eval_step(input, target):
+    output = MODEL(input)
+    loss = CRITERIA(output, target).item()
+    return float(loss), (output.argmax(1) == target).type(torch.float).sum().item()
+
+
+def evaluate():
+    MODEL.eval()
+    total_loss = 0
+    correct_predictions = 0
+    total_predictions = 0
+    counter = 0
+    with torch.no_grad():
+        for batch in DEV_DATAPIPE:
+            input = F.to_tensor(batch['token_ids'], padding_value=PADDING_IDX).to(DEVICE)
+            target = torch.tensor(batch['target']).to(DEVICE)
+            loss, predictions = eval_step(input, target)
+            total_loss += loss
+            correct_predictions += predictions
+            total_predictions += len(target)
+            counter += 1
+
+    return total_loss / counter, correct_predictions / total_predictions
+
+
+#######################################################################
+# Train
+# -----
+# TODO
+
+NUM_EPOCHS = 1
+
+for e in range(NUM_EPOCHS):
+    loss, accuracy = evaluate()
+    print("Epoch = [{}], loss = [{}], accuracy = [{}]".format(e, loss, accuracy))
+    MODEL.train()
+    for batch in TRAIN_DATAPIPE:
+        input = F.to_tensor(batch['token_ids'], padding_value=PADDING_IDX).to(DEVICE)
+        target = torch.tensor(batch['target']).to(DEVICE)
+        train_step(input, target)
+
+loss, accuracy = evaluate()
+print("Epoch = [{}], loss = [{}], accuracy = [{}]".format(NUM_EPOCHS), loss, accuracy)
```

Comment:
It might be a good idea to show the result of the training. like running an inference on couple of examples, if that can be done without adding too much.